components/WarningNote/WarningNoteRecap/WarningNoteRecap.tsx

```diff
@@ -24,8 +27,34 @@ const WarningNoteRecap = ({
     return <Spinner />;
   }
 
+  const displayReviews =
+    Array.isArray(warningNote.warningNoteReviews) &&
+    warningNote.warningNoteReviews.length > 0 &&
+    formName != 'Warning Note Created';
+
   return (
     <>
+      {displayReviews &&
+        warningNote.warningNoteReviews
+          .slice(0)
```

Comment:
What's the `.slice(0)` here doing?

Reply:
I saw this being used in a solution for returning an array in the reverse order.
My understanding is that `.slice()` would make a shallow copy of the array starting and ending from any indices provided. I'm not sure if it's entirely needed here, but I wondered if maybe having a shallow copy of the reviews array to manipulate was better than acting on the actual array. I might also be misunderstanding the intention behind having shallow copies and the `.slice()` function.

Reply:
Based on our conversation, I will be refactoring this to be more readable and clearer on its intent. 🚀 😁